python/translit/converters/translit.py

```diff
@@ -0,0 +1,22 @@
+"""
+Basic transliterator interface
+"""
+
+
+class Transliterator:
+
+    def convert(self, text):
+        """
+        Function responsible for conversion
+        :param text: String text to convert
+        :return: String conversion result
+        """
+        raise NotImplementedError("Need to be implemented")
+
+    def meta(self):
```

Comment:
That's a cool idea to have function like that.

Reply:
To be honest, I am not sure we really need class based implementation.
May be better to have dict key: Conversion direction and value: reference to function? 
What do you think @nbeloglazov ?

Because you are right here: https://github.com/belaudiobooks/lacinka/pull/2#discussion_r1264603385
I do not use this function.

Reply:
Yeah. Starting with individual functions sounds good. Later should be trivial enough to refactor to interface-like approach as long as it doesn't affect public API.